vnext/template/cpp-lib/proj/MyLib.vcxproj

```diff
@@ -18,13 +18,12 @@
     <AppContainerApplication>true</AppContainerApplication>
     <ApplicationType>Windows Store</ApplicationType>
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
-    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.19041.0</WindowsTargetPlatformVersion>
-    <WindowsTargetPlatformMinVersion>10.0.16299.0</WindowsTargetPlatformMinVersion>
   </PropertyGroup>
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Label="ReactNativeWindowsProps">
     <ReactNativeWindowsDir Condition="'$(ReactNativeWindowsDir)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(SolutionDir), 'node_modules\react-native-windows\package.json'))\node_modules\react-native-windows\</ReactNativeWindowsDir>
   </PropertyGroup>
+  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.WindowsSdk.Default.props" />
```

Comment:
An open question for both @asklar and @jonthysell 

Is it a goal for the lib template of version n to support older apps? I know the 0.64 template only supports 0.64 apps, but this change assumes prop sheets are present that are only in 0.66.

We could conditionally include them and include fallbacks, but that is unnecessary complexity if we ever go rely on another external property sheet (e.g. like we rely on WinUI.props being present in the 0.64 template).

Reply:
I would expect that new templates gracefully degrade when used with older dependencies - IOW, you might not get 100% of the functionality provided by the new template (e.g. overriding properties), but it still builds. So yes, it'd be a priority for new libraries to work with older apps, and vice versa.

Reply:
Added fallback in https://github.com/microsoft/react-native-windows/pull/8470/commits/94c9df856947c2d04964f4844df69772a6de9ea7